features/base/conference/reducer.js

```diff
@@ -0,0 +1,25 @@
+import { CONNECTION_DISCONNECTED } from '../connection';
+import { ReducerRegistry } from '../redux';
+
+import {
+    CONFERENCE_JOINED,
+    CONFERENCE_LEFT
+} from './actionTypes';
+
+/**
+ * Listen for actions that contain the conference object, so that
+ * it can be stored for use by other action creators.
+ */
+ReducerRegistry.register('features/base/conference', (state = null, action) => {
+    switch (action.type) {
+    case CONFERENCE_JOINED:
+        return action.conference;
+
+    case CONNECTION_DISCONNECTED:
+    case CONFERENCE_LEFT:
```

Comment:
This is more like a safe check to ensure that when CONNECTION_DISCONNECTED we clear the conference too. Also until my PR https://github.com/jitsi/lib-jitsi-meet/pull/161 was not merged CONFERENCE_LEFT was not fired at all, so this was a bit of workaround how to clean the conference.
But yes, I agree, we can remove CONNECTION_DISCONNECTED case here.